plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/condition/ShellMatchingMatcherIsAvailable.java

```diff
@@ -25,38 +26,49 @@
  */
 public class ShellMatchingMatcherIsAvailable extends AbstractWaitCondition {
```

Comment:
Do we need 2 matchers doing almost same thing? ShellMatchingMatcherIsAvailable is super of ShellWithTextIsAvailable. Plus we have ShellIsAvailable. We could have
public ShellIsAvailable(String title) 
public ShellIsAvailable(Matcher... matchers)
public ShellIsAvailable(Shell shell)
which would do all thing, we would avoid various names for wait conditions doing same stuff and would have it simplified.